lib/config/definitions.ts

```diff
@@ -12,7 +12,7 @@ const options: RenovateOptions[] = [
     description: 'If true allow templating for post-upgrade commands.',
     type: 'boolean',
     default: false,
-    admin: true,
+    globalOnly: true,
```

Comment:
We need to update website generator for this?

Reply:
Yes, part of why I put this in major so that we don't upgrade to it accidentally